go.mod

```diff
@@ -14,8 +14,8 @@ require (
 	github.com/jedib0t/go-pretty v4.3.0+incompatible
 	github.com/k0sproject/dig v0.2.0
 	github.com/k0sproject/k0s v1.29.6-0.20240527072442-22f6a125e881
-	github.com/replicatedhq/embedded-cluster-kinds v1.3.2
-	github.com/replicatedhq/embedded-cluster-operator v0.33.2
+	github.com/replicatedhq/embedded-cluster-kinds v1.3.3
+	github.com/replicatedhq/embedded-cluster-operator v0.34.2-0.20240531145409-ed39edc4db44
```

Comment:
TODO: change this once https://github.com/replicatedhq/embedded-cluster-operator/pull/196 is merged and operator is released